src/main/java/org/b612foundation/adam/opm/KeplerianCovariance.java

```diff
@@ -0,0 +1,183 @@
+package org.b612foundation.adam.opm;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Keplerian State Covariance Matrix (6x6 Lower Triangular Form). This is a custom extension to the
+ * OPM standard in  https://public.ccsds.org/Pubs/502x0b2c1.pdf . It should be serialized as
+ * USER_DEFINED_* terms per the standard.
+ */
+
+@EqualsAndHashCode
+@Getter
+@ToString
+public class KeplerianCovariance {
+  /** Reference frame. If omited, same as metadata */
+  private OdmCommonMetadata.ReferenceFrame covRefFrame = null;
+  /**
+   * Covariance matrix. The variables are in order:
+   *  - semi-major axis (A) in km.
+   *  - eccentricity (E) which is unitless
+   *  - inclination (I) in degrees
+   *  - right ascension of the ascending node (O) in degrees
+   *  - argument of pericenter (W) in degrees
+   *  - true anomaly (T) or mean anomaly (M) in degrees
+   *
+   * If both the true and mean anomaly fields are set the result could be undefined behaviors
+   * by code using this.
+   */
+  private double CAA;
```

Comment:
what does this stand for?

Reply:
Each element corresponds to the covariance matrix. The key to the six parameters is in the documentation. The reference document is here: https://docs.google.com/document/d/1i35imjxg5E-vTpY_CRC09vLoab1xX_cPM_FgUEUXvHA/edit

Reply:
Ah, got it. Is this something that people who would use Keplerian covariance will know about? (I do realize that it's not really the easiest thing to document matrices in javadoc and am happy to link to some kind of public documentation)

Reply:
It couldn't hurt to put a full matrix representation in there. I can add it for the Cartesian one as well. I'll put it just under the key